source/infrastructure/lib/aws-instance-scheduler-lambda.ts

```diff
@@ -0,0 +1,159 @@
+import {LambdaToDynamoDB, LambdaToDynamoDBProps} from "@aws-solutions-constructs/aws-lambda-dynamodb";
+import {Aws, RemovalPolicy} from "aws-cdk-lib";
+import * as lambda from "aws-cdk-lib/aws-lambda";
+import * as cdk from "aws-cdk-lib";
+import * as dynamodb from "aws-cdk-lib/aws-dynamodb";
+import {Construct} from "constructs";
+import {Effect, PolicyStatement} from "aws-cdk-lib/aws-iam";
+export enum DeploymentType {
```

Comment:
Yes, an S3Deployment and CDKDeployment class would probably be better object orientation. However, if this works the way I think it will the difference will amount to 1 or 2 lines across the entire cdk code script and so clearly showing that the 2 deploys are identical except for these 2 lines seems more important to me 

Additionally, if S3 and CDK deployment styles begin diverging we won't be able to trust that tests against 1 apply to the other, so this somewhat less flexible design also serves to protect against such divergence

Reply:
Fair enough, that answers the question. But still, prefer using Garvit's cdk-helper way of doing this.